src/main/java/frc/robot/RobotContainer.java

```diff
@@ -302,33 +327,42 @@ private void configDriverFeedBack() {
             Commands.startEnd(
                 () -> signalingSubsystem.startIntakeSignal(),
                 () -> signalingSubsystem.endSignal()));
-  }
 
-  private Command speakerSource() {
-    return Commands.parallel(
-        shooterCommandFactory.intakeSource(), triggerCommandFactory.getLoadCommand());
+    // Intake sensor for driver feedback
+    // Tells driver when note is aquired
+    new TOFSensorSimple(3, 250)
+        .beamBroken()
+        .whileTrue(
+            Commands.startEnd(
+                () -> {
+                  OI.Driver.controller.setRumble(
+                      RumbleType.kBothRumble, OperatorConstants.RUMBLE_STRENGTH);
+                },
+                () -> {
+                  OI.Driver.controller.setRumble(RumbleType.kBothRumble, 0);
+                },
+                new Subsystem[0]));
```

Comment:
Delete the subsystem parameter, its optional so adding a random new Subsystem is unnecessary.